src/engraving/rendering/dev/chordlayout.cpp

```diff
@@ -1823,6 +1823,34 @@ void ChordLayout::layoutChords1(LayoutContext& ctx, Segment* segment, staff_idx_
                 }
             }
 
+            // Create space between ledger lines and stems
+            // Above stave
+            Note* topUpNote = upStemNotes.back();
+            Note* bottomDownNote = downStemNotes.front();
+            if (!bottomUpNote->chord()->noStem() && bottomUpNote->chord()->stem()) {
+                int bottomUpStemLen = bottomUpNote->chord()->stem()->length() / sp * 2;
+                int firstLedgerAbove = -2;
+                // Don't adjust unisons or 2nds as these move anyway
+                int sep = topUpNote->line() - bottomDownNote->line();
+                if (topDownNote->line() < -1 && topDownNote->line() < bottomUpNote->line() && sep > 1
+                    && bottomUpNote->line() - bottomUpStemLen <= firstLedgerAbove) {
+                    upOffset += 0.15 * sp;
```

Comment:
I see a hard-coded offset here, but the length of ledger lines can be varied using `Sid::ledgerLineLength`. Shouldn't that be taken into account?